tests/helpers/properties/application-adapter.js

```diff
@@ -0,0 +1,43 @@
+import $ from 'jquery';
```

Comment:
as far as I understand `$` here is exactly the same as `Ember.$`. Probably we should replace all the `'jquery'` imports with `'-jquery'`  in tests as well?...

Reply:
This is something I'm not familiar with -- what's the difference between importing from `jquery` and `-jquery`?

Reply:
We can't rely on `Ember.$` cause it can be empty in apps which don't use jquery. 
`ember-cli-page-object` adds its own jquery version in case of app's jquery absence:

https://github.com/san650/ember-cli-page-object/blob/master/index.js#L29-L31

We call this jquery module '-jquery' in order to prevent overlapping with regular 'jquery'.
https://github.com/san650/ember-cli-page-object/blob/57183487aa9a8446861f4fbc52c03dbc02eeb1e6/vendor/shims/ecpo-jquery.js#L22

In case when `Ember.$` is present we also put it into '-jquery' for convenience:
https://github.com/san650/ember-cli-page-object/blob/57183487aa9a8446861f4fbc52c03dbc02eeb1e6/vendor/shims/project-jquery.js#L14

So I think `import from '-jquery'` is recommended way to use jquery inside this project.

---

I'm not sure if we have to change all the `jquery` imports to `-jquery` in  the tests at least until `Ember` drops `jquery` support. But we definitely have to do it in the [public code](https://github.com/san650/ember-cli-page-object/pull/357/files#diff-6940e0eec69729b817c170c2aa6fccc0R1).